components/Dialogs/src/MDCAlertController.m

```diff
@@ -282,8 +286,20 @@ - (void)setCornerRadius:(CGFloat)cornerRadius {
 }
 
 - (void)setElevation:(MDCShadowElevation)elevation {
+  BOOL shouldNotifyChanges = !MDCCGFloatEqual(elevation, _elevation);
   _elevation = elevation;
   self.mdc_dialogPresentationController.dialogElevation = elevation;
+  if (shouldNotifyChanges) {
+    [self.view mdc_elevationDidChange];
+  }
+}
+
+- (CGFloat)mdc_currentElevation {
+  return self.elevation;
+}
+
+- (CGFloat)mdc_overrideBaseElevation {
```

Comment:
This function should be removed as it's autogenerated by `@synthesize`

Reply:
Ah, yes, sorry. I put that there for debugging.